tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -341,189 +49,18 @@ info cli using local version of lerna
-@fluentui/docs:   Types of property 'design' are incompatible.
-@fluentui/docs:     Type 'Requireable<InferProps<{ position: Requireable<string>; display: Requireable<string>; top: Requireable<string>; right: Requireable<string>; bottom: Requireable<...>; ... 16 more ...; maxHeight: Requireable<...>; }>>' is not assignable to type 'Validator<ComponentDesignProp>'.,/office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Tree/TreeItem.tsx
-@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Tree/TreeItem.tsx(243,1):
-@fluentui/docs: TS2322: Type '{ contentRef: PropTypes.Requireable<((...args: any[]) => any) | { current: any; }>; id: PropTypes.Validator<string>; index: PropTypes.Requireable<number>; ... 19 more ...; accessibility: PropTypes.Requireable<...>; }' is not assignable to type 'WeakValidationMap<WithAsProp<TreeItemProps>>'.
-@fluentui/docs:   Types of property 'design' are incompatible.
-@fluentui/docs:     Type 'Requireable<InferProps<{ position: Requireable<string>; display: Requireable<string>; top: Requireable<string>; right: Requireable<string>; bottom: Requireable<...>; ... 16 more ...; maxHeight: Requireable<...>; }>>' is not assignable to type 'Validator<ComponentDesignProp>'.,/office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx
-@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx(129,1):
-@fluentui/docs: TS2322: Type '{ hasSubtree: PropTypes.Requireable<boolean>; index: PropTypes.Requireable<number>; level: PropTypes.Requireable<number>; onClick: PropTypes.Requireable<(...args: any[]) => any>; ... 10 more ...; accessibility: PropTypes.Requireable<...>; }' is not assignable to type 'WeakValidationMap<WithAsProp<TreeTitleProps>>'.
-@fluentui/docs:   Types of property 'design' are incompatible.
-@fluentui/docs:     Type 'Requireable<InferProps<{ position: Requireable<string>; display: Requireable<string>; top: Requireable<string>; right: Requireable<string>; bottom: Requireable<...>; ... 16 more ...; maxHeight: Requireable<...>; }>>' is not assignable to type 'Validator<ComponentDesignProp>'.,/office-ui-fabric-react/packages/utilities/src/object.ts
-@fluentui/docs: ERROR in /office-ui-fabric-react/packages/utilities/src/object.ts(15,42):
-@fluentui/docs: TS2536: Type 'Extract<keyof TA, string>' cannot be used to index type 'TB'.
-@fluentui/docs: Details:
-@fluentui/docs:     domainThrown: true
-@fluentui/docs: [XX:XX:XX] 'build:docs' errored after 
-@fluentui/docs: error Command failed with exit code 1.
-lerna ERR! yarn run build exited 1 in '@fluentui/docs'
+@fluentui/keyboard-key: Warning: You have changed the public API signature for this project. Please copy the file "temp/keyboard-key.api.md" to "etc/keyboard-key.api.md", or perform a local build (which does this automatically). See the Git repo documentation for more info.
```

Comment:
I bet this is a result of a d.ts emit change. But I'm not sure how to accept the change without squashing errors from this project. Maybe submitting a PR to office-ui-fabric is the right thing.

@weswigham can you figure out what to do here?

Reply:
I've just been accepting the message that it changed - once they pull in a version of TS the generates the same declarations upstream, the message'll go away and we'll accept it again.